### PR TITLE
Bellatrix spec tests

### PR DESCRIPTION
**Pull request by terencechain**
Bringing down merge spec tests from `kintsugi` branch to `develop` branch.

### CODE REVIEW

**Comment by terencechain** [comment]
Screen Shot 2022-01-07 at 9 10 40 AM

looks funny hehe

File: testing/spectest/shared/bellatrix/ssz_static/ssz_static.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+// RunSSZStaticTests executes "ssz_static" tests.
+func RunSSZStaticTests(t *testing.T, config string) {
+	require.NoError(t, utils.SetConfig(t, config))
+
+	testFolders, _ := utils.TestFolders(t, config, "bellatrix", "ssz_static")
+	for _, folder := range testFolders {
+		innerPath := path.Join("ssz_static", folder.Name(), "ssz_random")
+		innerTestFolders, innerTestsFolderPath := utils.TestFolders(t, config, "bellatrix", innerPath)
+		for _, innerFolder := range innerTestFolders {
+			t.Run(path.Join(folder.Name(), innerFolder.Name()), func(t *testing.T) {
```
Comment (prestonvanloon) [inline testing/spectest/shared/bellatrix/ssz_static/ssz_static.go:34]: Please see PR #10003 and add the same changes to support the different ssz static modes.

Comment (terencechain) [reply testing/spectest/shared/bellatrix/ssz_static/ssz_static.go]: Done. Thanks for the feedback

**Comment by prestonvanloon** [comment]
gofmt please

**Pull request merged**